pyfstat/helper_functions.py

```diff
@@ -628,3 +628,164 @@ def get_parameters_dict_from_file_header(outfile, comments="#", eval_values=Fals
             val = eval(val)  # DANGER
         params_dict[key] = val
     return params_dict
+
+
+def read_par(
```

Comment:
But I see you decided to keep this one and not the other one...

Reply:
@dbkeitel Are you really keeping this function here just to do a test?
I thought Test classes were derived from BaseSearch so this function can be accessed as an inherited method...?

Reply:
See above for why it's a stand-alone function.

The test could indeed be integrated better, however I think we've spent enough time on test cleanups for now. ;)

Reply:
No problem: It's a stand-alone function, it has a stand-alone test.